### ISSUE TITLE
What about TypedArray views?

### ISSUE THREAD

**Issue by ghost** [keithamus/proposal-array-last#18]
The rationale would seem to apply equally to them.

**Comment by ljharb** [comment]
Good point; anything that is added here to Array.prototype should probably be added to TypedArray.prototype.

**Comment by keithamus** [comment]
PRs always welcome 👍

**Comment by zloirock** [comment]
2 questions: should proposed accessors validate that the target is a typed array and what the behaviour expected for typed arrays with zero length?

**Comment by addaleax** [comment]
@zloirock Is there any reason to do it differently than for `Array`s? As far as I can tell, the [proposed spec](https://github.com/keithamus/proposal-array-last#specification) would work for typed arrays without changes… right?

**Comment by zloirock** [comment]
@addaleax almost all typed array methods are non-generic and throws an error on something different from typed array. Typed arrays have a constant length, so setting `.lastItem` for typed arrays with zero length by the current spec is an error.

**Comment by jhpratt** [comment]
If the issue @zloirock brought up is resolved, I can look into submitting a PR to add `.lastItem` and `.lastIndex` for TypedArrays.
